door-open-service/node-backend/src/slackInterface.js

```diff
@@ -23,6 +23,19 @@ export default function (config, slackBot, sockServer) {
           "Consider dispatching a drone to pick up a human.");
 
     } else {
+
+      config.doorTimes = { "garage": 25 * 1000 };
+
+      if (config.lock) {
```

Comment:
I think it's preferable to store the lock elsewhere. The lock isn't something that should belong to this object, and for the purpose of this application, `config` is handled as a read-only object.